CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,81 @@
+The number one risk to Open Source projects is maintainer burnout, and the number one cause of maintainer burnout is impoliteness in various forms, usually from a minority of users.
+
+## Politeness
+
+It is unfortunately common in Open Source projects for a minority of users act with aggression and disrespect (e.g. "How dare you not fix this already"), but we do not accept it here.
+
+We don't expect too much in terms of overt politeness - simply don't be impolite.
+You can post a question, get an answer, and then never say thank you - that's totally fine!
+But do not respond back with something like "Pity the documentation didn't say that in the first place" or "thanks for nothing", etc.
+
+## Respect the time of those who help you
+
+Respect is bidirectional, but unfortunately time is not an equal resource and those seeking help need to understand that maintainer time is limited.
+We may see hundreds of different users/questions each week, and try to help every one of them.
+In order to "scale" this support, responsibility is usually placed on the user to provide logs or reproductions of their problem.
+If you're not willing to accept this, you are unlikely to get the support you're requesting.
+
+It's also important to understand that the majority of time you see spent supporting users is voluntary and not paid - even by Mend.io staff.
+
+## Blocking and unblocking
+
+We act quickly and decisively to address impoliteness from community members through automated comments, or temporary or permanent banning.
+If you display repeated undesirable behavior (e.g. spamming maintainers with @mentions) or state openly your defiance to the rules ("How dare you ask me not to @mention you?") then you'll receive a block.
+We do not engage in "debate" about such blocks, but we'll almost always undo a block for any user who demonstrates a newfound understanding of the rules or explains it was a misunderstanding.
+
+In other words: we can be quick to both block and unblock, and all that matters is that you can conduct yourself in future according to the project's rules.
+
+## How we prioritize work
+
+When it comes to Renovate's core contributors and maintainers, we prioritize features and fixes:
+
+- those affecting a large number of users, or
+- regression errors (something used to work, but we broke it by accident), or
+- features needed by a Mend.io customer (Renovate's sponsor/owner), or
+- are sponsored by 3rd parties after independent validation ( PRs will not be accepted if they are not in the interest of Renovate OSS ), or
+- those we need personally, or just want to see done
+
+A surprising number of users find it hard to accept that the Renovate core team or other community contributors won't build every feature or fix every bug which they'd like to see.
+Participating in this repo is dependent on you finding a way to accept this reality, and not being pushy about it.
```

Comment:
```suggestion
When users are excited or pressured to get an implementing working then it can be hard to accept that there's nobody to add their particular feature or look into their bug, but it's important that you understand and accept this reality.
```